.github/workflows/main.yml

```diff
@@ -14,8 +14,9 @@ jobs:
       with:
         ocaml-compiler: ${{ matrix.ocaml-compiler }}
     - run: opam pin -n .
-    - run: opam depext -yt tiny_httpd tiny_httpd_camlzip
-    - run: opam install -t . --deps-only
-    - run: opam exec -- dune build @install
-    - run: opam exec -- dune runtest
-      if: ${{ matrix.operating-system == 'unbuntu-latest' }}
```

Comment:
I removed this `if` clause because it can be independently disabled directly in the dune files if necessary via `(enable_if (= %{os} "linux"))`

Reply:
you should revert though, I forgot, there's a depext that doesn't work on windows apparently (curl)

Reply:
then the proper fix would be to add `"conf-curl" {with-test & os != "win32"}`

Reply:
but then the test on win32 will fail? so they still need to be disabled in CI

Reply:
not if you use `(enable_if (<> %{os} "win32"))`